desktop/app/app.go

```diff
@@ -275,28 +275,24 @@ func (app *App) Run(isMain bool) {
 	}()
 }
 
-// setFeatures enables or disables the features specified by values in the features map
-// sent back to the UI
-func (app *App) setFeatures(enabledFeatures map[string]bool, values map[features.Feature]bool) {
-	for feature, isEnabled := range values {
-		if isEnabled {
-			enabledFeatures[feature.String()] = isEnabled
-		}
-	}
-}
-
 // checkEnabledFeatures checks if features are enabled
-// (based on the env vars at build time or the user's settings/geolocation)
-// and starts appropriate services
 func (app *App) checkEnabledFeatures() {
 	enabledFeatures := app.flashlight.EnabledFeatures()
-
-	app.setFeatures(enabledFeatures, features.EnabledFeatures)
-
 	log.Debugf("Starting enabled features: %v", enabledFeatures)
 	//go app.startReplicaIfNecessary(enabledFeatures)
 }
 
+// IsFeatureEnabled checks whether or not the given feature is enabled by flashlight
+func (app *App) IsFeatureEnabled(feature string) bool {
```

Comment:
@atavism Since this is a map, Can we do something like this?

```
func (app *App) IsFeatureEnabled(feature string) bool {
	enabledFeatures := app.flashlight.EnabledFeatures()[feature]
	return enabledFeatures
}
```

Reply:
Oh, definitely! Thanks. Just made that change.